### PR TITLE
Update TransactionsNavigationHeader.tsx

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: components/TransactionsNavigationHeader.tsx [hunk 1]
```diff
@@ -296,7 +296,7 @@ const styles = StyleSheet.create({
   walletBalance: {
     flexShrink: 1,
     marginRight: 6,
-    height: 34,
+    marginT: 50,
```
Comment (Overtorment) [inline components/TransactionsNavigationHeader.tsx:299]: Typo..?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: components/TransactionsNavigationHeader.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/TransactionsNavigationHeader.tsx]: fixed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2m5lbgazmc3ed5t4x2ivs7r2me?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1283)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xulyh3rgzwsgrli3h3b3qtkjfm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1284)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**